lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 632 tooltips

## Description

Add tooltips to disabled buttons. I added them wherever I could find them, it could be that I missed one or two here or there.
There are three options for the tooltips, left aligned, center aligned and right aligned. 
![Screenshot 2024-10-03 at 08 59 47](https://github.com/user-attachments/assets/f285401f-102f-417d-a19f-26bf3197a1b6)
![Screenshot 2024-10-03 at 09 35 30](https://github.com/user-attachments/assets/276b568b-5840-4833-ad69-14afa795a2c8)
![Screenshot 2024-10-03 at 09 35 47](https://github.com/user-attachments/assets/e039e8df-db05-491e-8c60-b07e46fd3ade)

## How to test

Find some disabled tooltips and see if they look okay and render in the correct place.
